source/versioning.rst

```diff
@@ -3,15 +3,16 @@
 3-D Secure version determination
 ================================
 
-There are currently three 3-D Secure versions, ``1.0.2``, ``2.1.0`` and ``2.2.0``.
+There are currently four 3-D Secure versions, ``1.0.2``, ``2.1.0``, ``2.2.0`` and ``2.3.1.1``.
```

Comment:
> There are currently four 3-D Secure versions, ``1.0.2``, ``2.1.0``, ``2.2.0`` and ``2.3.1.1``.

We don't support 1.0.2 and as far as I know this is not supported by anyone anymore.

```suggestion
There are currently three 3-D Secure versions, ``2.1.0``, ``2.2.0`` and ``2.3.1.1``.
```

Reply:
Agree